### ISSUE TITLE
Typo in doc for "role-support-aria-props"

### ISSUE THREAD

**Issue by Mintoo200** [jsx-eslint/eslint-plugin-jsx-a11y#788]
In the [documentation page for "role-supports-aria-props"](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/role-supports-aria-props.md), in the `rule details` section, the `Succeed` subpart is correct with:
```html
<!-- Good: the radiogroup role does support the aria-required property -->
<ul role="radiogroup" aria-required aria-labelledby="foo">
...
```
but the `Fail` subpart has a typo:
```html
<!-- Bad: the radio role does not support the aria-required property -->
<ul role="radiogroup" aria-labelledby="foo">
...
```
Notice the `role="radiogroup"` when the comment says `radio role`.

**Comment by Mintoo200** [comment]
Hey, I never participated in open source project but I'd be willing to give it a go if anyone could point me toward the process to submit a merge request?

**Comment by ljharb** [comment]
Since this is purely documentation, you can initiate a pull request directly in the web interface. navigate to the page (https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/role-supports-aria-props.md) and click the "pencil" icon. Hopefully the flow will be self-explanatory.

**Comment by Mintoo200** [comment]
Actually the documentation is right, my mistake: the comment applies to the `<li>` which has role "radio", not the `<ul>`.
Sorry for the inconvenience.
